### PR TITLE
First opam version for bindlib

### PR DESCRIPTION
**Pull request by craff**

### CODE REVIEW

File: packages/bindlib/bindlib.4.0/descr [hunk 1]
```diff
@@ -0,0 +1,10 @@
+OCaml Bindlib library for bound variables
+
+Bindlib is a library for the Objective-Caml language providing
```
Comment (lefessan) [inline packages/bindlib/bindlib.4.0/descr]: "Objective-Caml" does not exist any more, please use "OCaml"

**Comment by lefessan** [comment]
Could you squash all commits in one (`git rebase -i master` and then replace all `pick` by `s` except the first one) ? 10 commits to add 3 files is a bit too much :-)

**Comment by craff** [comment]
I did the rebase ... how to I "replace all pick by s" ... I am really no git expert !

**Comment by lefessan** [comment]
Normally, if you use "git rebase -i master", git will open your favorite editor, with a file containing one commit per line, preceeded by the "pick" word. You should then replace all these "pick" by "s" (for squash), except the first line, save and close, then git will ask you for the final commit message, and you are done.

**Comment by craff** [comment]
Comme j'ai déjà fait (par erreur) git rebase ... Je n'arrive plus à voir mes patchs (vive darcs)

**Comment by lefessan** [comment]
Maybe you can just restart from a clean master, add the final 3 files of this PR, and push -f to replace it ?

**Comment by craff** [comment]
and do like in https://xkcd.com/1597/ ;-)

**Comment by lefessan** [comment]
Exactly ;-) 
Basically, you should never commit in your master. Your master should always be a clean copy of the main repository master, and your commits should always go into branches. If you follow this rule, you will never need to clone again.

**Comment by craff** [comment]
I did not clone ...

**Comment by Chris00** [comment]
On 2016-05-30, Christophe Raffalli wrote:
> 
> Comme j'ai déjà fait (par erreur) git rebase ... Je n'arrive plus à 
> voir
> mes patchs (vive darcs)

« git reflog » peut aider.  Rien n'est perdu sous Git tant qu'on ne fait 
pas « git gc » ou qu'il n'est pas fait auto.

Cordialement,
C.

**Pull request closed without merging**